tutorials/toolkits/axisartist.py

```diff
@@ -541,7 +541,7 @@ def inv_tr(x, y):
     # system in degree
     tr = Affine2D().scale(np.pi/180., 1.) + PolarAxes.PolarTransform()
 
-    # extreme finder :  find a range of coordinate.
+    # extreme finder: find a range of coordinate.
```

Comment:
I would not change this one occurence. There seems to be some consistency in having a space before the colon throughout the tutorial. Either change all or none.

Reply:
they're all changed now to no space (double colons at end of paragraphs introducing code blocks don't show up in the rendered html so are irrelevant here).